### PR TITLE
Disable inheritance margin for interactive documents

### PR DESCRIPTION
**Pull request by dibarbet**
This 'fixes' 2 out of 3 debug assertions thrown in https://github.com/dotnet/roslyn/issues/61426 when the interactive window is opened.  One is still thrown during incremental analysis and still needs a real fix, so am leaving the issue open

It isn't a real fix, but regardless we shouldn't be doing inheritance margin tagging work for interactive documents since there is no margin shown.

### CODE REVIEW

**Review by davidwengier — APPROVED** [review]

File: src/VisualStudio/Core/Def/InheritanceMargin/InheritanceMarginTaggerProvider.cs [hunk 1]
```diff
@@ -95,6 +96,9 @@ protected override IEnumerable<SnapshotSpan> GetSpansToTag(ITextView? textView,
             if (document == null)
                 return;
 
+            if (document.Project.Solution.Workspace.Kind == WorkspaceKind.Interactive)
```
Comment (CyrusNajmabadi) [inline src/VisualStudio/Core/Def/InheritanceMargin/InheritanceMarginTaggerProvider.cs:99]: probably worth a comment.  but no biggie.

**Review by CyrusNajmabadi — APPROVED** [review]

**Review by Cosifne — APPROVED** [review]

**Comment by jasonmalinowski** [comment]
> we shouldn't be doing inheritance margin tagging work for interactive documents since there is no margin shown.

I would have assumed if there's no margin, then there won't be anybody creating our tagger in the first place -- did we go sideways somewhere else?

**Pull request merged**